src/controllers/airtimeControllers.js

```diff
@@ -0,0 +1,38 @@
+require('dotenv').config();
```

Comment:
Let's change the name of the the controller to `transactionControllers` so all transfer for funds, airtime and data will be under it

Reply:
You can call the function for the airtime transfer controller `transferAirtime` or something like that